docs/kcl-lang/numeric.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Numeric types and units"
+excerpt: "Documentation of the KCL language for the Zoo Design Studio."
+layout: manual
+---
+
+Numbers and numeric types in KCL include information about the units of the numbers. So rather than just having a number like `42` we always have information about the units so we don't confuse 42 mm with 42 inches.
```

Comment:
```suggestion
Numbers and numeric types in KCL include information about the units of the numbers. So rather than just having a number like `42`, we always have information about the units so we don't confuse 42 mm with 42 inches.
```